src/react/index.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import octicons from '../octicons'
```

Comment:
Property `icon` will be an object, not a string (name). Therefore, we don't need to include the octicons here. See my suggestions: https://github.com/hiendv/octicons-modular/issues/6#issuecomment-335081996